core/frontend/package.json

```diff
@@ -61,6 +61,7 @@
     "@itwin/core-quantity": "workspace:*",
     "@itwin/certa": "workspace:*",
     "@itwin/eslint-plugin": "workspace:*",
+    "@itwin/extension-client": "^0.1.1",
```

Comment:
should probably be a direct dep

Reply:
We should not be adding the extension client as a direct dependency of core-frontend.  The ServiceExtensionProvider should be moved to the extension client package directly so that people can register the provider.

Reply:
Just to clarify, just ServiceExtensionProvider should be completely removed from core-frontend?

EmbedAPIHandler needs to be updated too, to get the service provider from the client package instead of core-frontend.

Reply:
Should I close this pr? Now that ServiceExtensionProvider has been moved to the extension-client package.

Reply:
no dont close this pr, you should remove the ServiceExtensionprovider from here altogether now